src/alerts/flash.scss

```diff
@@ -86,7 +86,7 @@
   }
 }
 
-.flash-warn {
+.flash-warn:not(.Banner--warning) {
```

Comment:
It maybe sufficient enough here to include `:not(.Banner)` on all of these instead of the scheme but could also be fine to include this as the Banner scheme also if you think there's a case where someone would have `flash-*` with `Banner--*` but not have `Banner`

Reply:
Hmm interesting. I don't think that would happen? Let's go with `:not(.Banner)` for now.